applications/Chat/coati/dataset/reward_dataset.py

```diff
@@ -1,54 +1,16 @@
-from typing import Callable
+from typing import Callable, Dict
 
 from torch.utils.data import Dataset
 from tqdm import tqdm
 
-from .utils import is_rank_0
+from colossalai.logging import get_dist_logger
 
+from .utils import is_rank_0, read_string_by_schema
 
-# Dahoas/rm-static
-class RmStaticDataset(Dataset):
-    """
-    Dataset for reward model
-
-    Args:
-        dataset: dataset for reward model
-        tokenizer: tokenizer for reward model
-        max_length: max length of input
-        special_token: special token at the end of sentence
-    """
+logger = get_dist_logger()
```

Comment:
This should be a local variable. Please avoid using global variable if unnecessary.